lib/tasks/fincap.rake

```diff
@@ -0,0 +1,25 @@
+namespace :fincap do
```

Comment:
Could we not have used `db/seed.rb` to seed these files?

Reply:
The cms db/seeds.rb is used mainly for development but this rake task should be run in production as well so we choose a rake task because then the code will not have data creation altogether CMS and Fincap.

Reply:
It's unconventional to create data using a rake task especially when you can seed production database with `RAILS_ENV=production bundle exec rails db:seed`. Anyway if you are convinced this is the best approach then go for it.

Reply:
Sorry, perhaps I was not clear. The cms db/seeds.rb has a lot of data from our **current CMS**. **The Fincap CMS will not have the same data**, so run the seeds as you described will not solve the problem in any way I can see it. So that's the reason that we didn't added to the seeds nor any other method than a rake task. Please, let's talk in person or via other method for clarification on this.